gateway/Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN cargo install --locked --path .
 FROM chef AS testing_env
 RUN rustup component add llvm-tools && \
     rustup component add rustfmt && \
-    cargo install grcov
+    cargo install --locked grcov && \
+    cargo install --locked taplo-cli
```

Comment:
I don't think it's still needed

Reply:
Indeed, thanks for catching that. Fixed in [c3e350b](https://github.com/osrd-project/osrd/pull/7053/commits/c3e350b24ade6ede3c5714c86414f6bfcbdbfe35).